### PR TITLE
2587 fix universe

### LINKED ISSUE #2587: EOrust.java:137-145: Fix EOrust. After getting rid of...
The puzzle `2437-10dd7966` from #2437 has to be resolved: 

https://github.com/objectionary/eo/blob/1325da69ced10e1fb5869424d08970c678848abb/eo-runtime/src/main/java/EOorg/EOeolang/EOrust.java#L137-L145

The puzzle was created by rultor on 14-Nov-23. 

Estimate: 60 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by levBagryansky**
Closes #2587

---

## PR-Codex overview
This PR focuses on fixing issues with the EOrust class and updating the UniverseDefaultTest class in the eo-runtime module.

### Detailed summary
- Fixed an issue in EOrust related to the "code" attribute locator.
- Updated the putsToCopy method in UniverseDefaultTest to use the "Q.org.eolang.bytes" type instead of "Q.org.eolang.int".
- Updated the bindsCopyToAbstract method in UniverseDefaultTest to use the "Q.org.eolang.bytes" type instead of "Q.org.eolang.int".
- Updated the rust-tests.eo file in the eo-runtime module.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by levBagryansky** [comment]
@maxonfjvipon please take a look

File: eo-runtime/src/test/java/org/eolang/UniverseDefaultTest.java [hunk 1]
```diff
@@ -147,11 +147,10 @@ void copies() {
     }
 
     @Test
-    @Disabled
     void putsToCopy() {
         final Map<Integer, Phi> indexed = new HashMap<>();
         final Universe universe = new UniverseDefault(Phi.Φ, indexed);
-        final int eoint = universe.find("Q.org.eolang.int");
+        final int eoint = universe.find("Q.org.eolang.bytes");
```
[the remark this answers is not in the document]
Comment (levBagryansky) [reply eo-runtime/src/test/java/org/eolang/UniverseDefaultTest.java]: @maxonfjvipon done

File: eo-runtime/src/test/java/org/eolang/UniverseDefaultTest.java [hunk 2]
```diff
@@ -163,12 +162,11 @@ void putsToCopy() {
     }
 
     @Test
-    @Disabled
     void bindsCopyToAbstract() {
         final Phi dummy = new DummyAbstract(Phi.Φ);
         final Map<Integer, Phi> indexed = new MapOf<>(dummy.hashCode(), dummy);
         final Universe universe = new UniverseDefault(dummy, indexed);
-        final int eoint = universe.find("Q.org.eolang.int");
+        final int eoint = universe.find("Q.org.eolang.bytes");
```
[the remark this answers is not in the document]
Comment (levBagryansky) [reply eo-runtime/src/test/java/org/eolang/UniverseDefaultTest.java]: @maxonfjvipon done

**Review by maxonfjvipon — APPROVED** [review]
@levBagryansky LGTM! Thanks

**Comment by levBagryansky** [comment]
@yegor256 please check

**Comment by levBagryansky** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**